pallets/subtensor/src/macros/errors.rs

```diff
@@ -22,33 +22,42 @@ mod errors {
         HotKeyAccountNotExists,
         /// The hotkey is not registered in any subnet.
         HotKeyNotRegisteredInNetwork,
-        /// Request to stake, unstake or subscribe is made by a coldkey that is not associated with the hotkey account.
+        /// Request to stake, unstake or subscribe is made by a coldkey that is not associated with
+        /// the hotkey account.
         NonAssociatedColdKey,
         /// The hotkey is not a delegate and the signer is not the owner of the hotkey.
         HotKeyNotDelegateAndSignerNotOwnHotKey,
         /// Stake amount to withdraw is zero.
         StakeToWithdrawIsZero,
-        /// The caller is requesting removing more stake than there exists in the staking account. See: "[remove_stake()]".
```

Comment:
Do we need all these lint changes ? We should be using the exisitng defaults

Reply:
This was actually not lint, but me - having comments which span across the line width is ugly and inconsistent. I think we should actually configure the formatter to handle it.

Reply:
For context (DMd to @distributedstatemachine about it). Rust formatter configured with the line width of 100 characters by default. The comments should not be exception, and it would be great to configure the formatter to take comments into account as well.